plugins/aver-hid/aver-hid.quirk

```diff
@@ -10,3 +10,8 @@ Plugin = aver_hid
 [USB\VID_2574&PID_0AB1]
 Plugin = aver_hid
 Flags = dual-isp
+
+# AVer CAM340Plus
+[USB\VID_2574&PID_0980]
+Plugin = aver_hid
+Flags = safe-isp
```

Comment:
is safe-isp a different operating mode of the existing protocol, or a different protocol altogether? I wonder if it would be less confusing to just have a fu-aver-safeisp-device.c instead.

Reply:
Yes, this is a different protocol. We will modify to fu-aver-safeisp-device.c.